zombies.js

```diff
@@ -178,7 +251,16 @@
  * @name eat
  * @param {Food} itemToEat  The food item to eat.
  */
-
+Player.prototype.eat = function (itemToEat) {
+  let indexOfPack = this.getPack().indexOf(itemToEat);
+  if (this.getPack().includes(itemToEat) && itemToEat instanceof Food && indexOfPack !== -1) {
```

Comment:
This .includes is not necessary since you're already checking to see if the indexOfPack !== -1